content/influxdb/cloud-dedicated/primers/api/v1/_index.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Use the InfluxDB v1 API with InfluxDB Cloud Dedicated (Single-tenant)
+list_title: Use the InfluxDB v1 API
+description: >
+  Use InfluxDB v1 API authentication, endpoints, and tools with InfluxDB Cloud Dedicated (Single-tenant).
+  Use the v1 API when you have existing v1 workloads that need it.
+weight: 3
+menu:
+  influxdb_cloud_dedicated:
+    parent: API primers
+    name: v1 API primer
+influxdb/cloud-dedicated/tags: [write, line protocol]
+---
+
+Use the InfluxDB v1 API when you have existing v1 workloads that need it.
+The v1 `/write` and `/query` endpoints work with username/password authentication, existing InfluxDB 1.x tools, and your custom code.
+
+{{% warn %}}
+{{% api/cloud/v2-prefer %}}
+{{% /warn %}}
+
+The InfluxDB v1 API `/write` endpoint works with
+InfluxDB 1.x client libraries and the Telegraf v1 Output Plugin.
+
+The InfluxDB v1 API `/query` endpoint uses SQL and works with Flight SQL libraries and plugins,
+and third-party integrations like [Grafana](https://grafana.com).
```

Comment:
The v1 endpoint still only supports InfluxQL. SQL is supported through the Flight SQL endpoint, with is all gRPC-based. At some point, you will be able to push both SQL and InfluxQL to the Flight SQL endpoint.

```suggestion
The InfluxDB v1 API `/query` endpoint supports InfluxQL and, and third-party integrations like [Grafana](https://grafana.com).
```